web/src/compose_validate.js

```diff
@@ -355,9 +356,18 @@ export function set_wildcard_mention_large_stream_threshold(value) {
     wildcard_mention_large_stream_threshold = value;
 }
 
-function validate_stream_message_mentions(stream_id) {
+export function validate_stream_message_mentions(
+    stream_id,
+    $banner_container,
+    from_typeahead = false,
+) {
     const subscriber_count = peer_data.get_subscriber_count(stream_id) || 0;
 
+    if (from_typeahead) {
+        // We will not update wildcard mention if we came from compose_typeahead
+        // We are sure that wildcard_mention exists, so we don't need to check it here.
+        wildcard_mention = "";
+    }
```

Comment:
What would happen if we didn't do this check?

Reply:
Also, we're not setting this value to `null`, so I'm confused -- won't the below check still run?

Reply:
In the first iteration, it helped to control the visibility of the banner. It was definitely a bug somewhere else, but I was able to fix it this way. Now this logic has been removed.